crates/egui/src/viewport.rs

```diff
@@ -799,6 +817,48 @@ pub enum WindowLevel {
     AlwaysOnTop,
 }
 
+#[derive(Clone, Copy, Debug, Default, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub enum WindowType {
```

Comment:
let's make it clear this is a X11 thing

```suggestion
pub enum X11WindowType {
```

Reply:
totally agree and done!